apps/demo/emails/apple-receipt.tsx

```diff
@@ -0,0 +1,528 @@
+import { Container } from '@react-email/container';
+import { Head } from '@react-email/head';
+import { Hr } from '@react-email/hr';
+import { Html } from '@react-email/html';
+import { Img } from '@react-email/img';
+import { Link } from '@react-email/link';
+import { Preview } from '@react-email/preview';
+import { Section } from '@react-email/section';
+import { Column } from '@react-email/column';
+import { Text } from '@react-email/text';
+import * as React from 'react';
+
+export default function Email() {
+  const baseUrl = process.env.VERCEL_URL
+    ? `https://${process.env.VERCEL_URL}`
+    : '';
+
+  return (
+    <Html>
+      <Head />
+      <Preview>Apple Receipt</Preview>
+      <Section style={main}>
+        <Container style={container}>
+          <Section>
+            <Column>
+              <Img
+                src={`${baseUrl}/static/apple-logo.png`}
+                width="42"
+                height="42"
+                alt="Apple Logo"
+              />
+            </Column>
+
+            <Column align="right" style={tableCell}>
+              <Text style={heading}>Receipt</Text>
+            </Column>
+          </Section>
+          <Section>
+            <Text style={cupomText}>
+              Save 3% on all your Apple purchases with Apple Card.
+              <sup style={supStyle}>1</sup>{' '}
+              <Link
+                href='href="https://www.apple.com/apple-card?rid=287-cid%3Dapy-120-100002"'
+                target="_blank"
+              >
+                Apply and use in minutes
+              </Link>
+              <sup style={supStyle}>2</sup>
+            </Text>
+          </Section>
+          <table style={informationTable}>
+            <tbody>
+              <tr style={informationTableRow}>
+                <td style={informationTableColumn} width={320} colSpan={2}>
```

Comment:
Instead of `table`, `body`, `tr`, `td`. Can you use our components?

Reply:
I tried during development, but I think there isn't a primitive in `react-email` to represent `tr`.
I might be wrong, but looking at the `Section` code, there isn't a way to add multiple `tr` to the component, so it might not be possible to use `Section` and `Column` for this use case.
Maybe there should exist a `Row` component 🤔